Mismatch result in `test_wsireader`

```
======================================================================
FAIL: test_resolution_mpp_0__home_runner_work_MONAI_MONAI_tests_testing_data_temp_wsi_generic_tiff_tiff (tests.test_wsireader.TestTiffFile.test_resolution_mpp_0__home_runner_work_MONAI_MONAI_tests_testing_data_temp_wsi_generic_tiff_tiff)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/parameterized/parameterized.py", line 620, in standalone_func
    return func(*(a + p.args), **p.kwargs, **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/MONAI/MONAI/tests/test_wsireader.py", line 594, in test_resolution_mpp
    self.assertTupleEqual(mpp, expected_mpp)
AssertionError: Tuples differ: (0.1, 0.1) != (1000.0, 1000.0)

First differing element 0:
0.1
1000.0

- (0.1, 0.1)
+ (1000.0, 1000.0)

----------------------------------------------------------------------
```

--- a/monai/data/wsi_reader.py
+++ b/monai/data/wsi_reader.py
@@ -1097,7 +1097,7 @@ class TiffFileWSIReader(BaseWSIReader):
         ):
             unit = wsi.pages[level].tags.get("ResolutionUnit")
             if unit is not None:
-                unit = str(unit.value)[8:]
+                unit = str(unit.value.name)
             if unit is None or len(unit) == 0:
                 warnings.warn("The resolution unit is missing. `micrometer` will be used as default.")
                 unit = "micrometer"

